ward/_debug.py

```diff
@@ -54,7 +55,7 @@ def _get_debugger_hook(hookname: str):
     modname, dot, funcname = hookname.rpartition(".")
     if dot == "":
         modname = "builtins"
-    module = importlib.import_module(modname)
+    module: Any = importlib.import_module(modname)
```

Comment:
I think this can be `types.ModuleType`

Reply:
I think line 60 will be a problem then (accessing `Pdb` attribute of `import pdb`. That said, maybe a `type: ignore` on line 60 is better than this.

Reply:
Ah, I see. You could possibly create a Protocol to handle this, but in all honesty it's hardly worth the effort.

Reply:
Yep and the Protocol then doesn't make much sense in the else clause where `hookname != "pdb.set_trace"`.

I think this is just dynamically typed code and should be ignored by static type checker.